src/driver/latin_hypercube_random_test_point_generator.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2021 The Private Cardinality Estimation Framework Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Latin Hypercube random test point generator."""
+
+from typing import Iterable
+from typing import List
+
+import numpy as np
+import pyDOE
+from wfa_planning_evaluation_framework.data_generators.data_set import DataSet
+from wfa_planning_evaluation_framework.driver.test_point_generator import (
+    TestPointGenerator,
+)
+from wfa_planning_evaluation_framework.driver.uniformly_random_test_point_generator import (
```

Comment:
I think the better approach would be to either define a constant MINIMUM_NUMBER_OF_TEST_POINTS that is unique to this class.  Alternatively, define this constant in the base class TestPointGenerator.  Then, both this class and UniformlyRandomTestPointGenerator can import this value from the base class.

Reply:
Makes sense. Now define MINIMUM_NUMBER_OF_TEST_POINTS in the base class TestPointGenerator.

(MAXIMUM_NUMBER_OF_TEST_POINTS is unique to Latin Hypercube)